src/Content/Conversation.php

```diff
@@ -442,16 +442,16 @@ public function create(array $items, $mode, $update, $preview = false, $order =
 					. "; var netargs = '" . substr($this->args->getCommand(), 8)
 					. '?f='
 					. (!empty($_GET['contactid']) ? '&contactid=' . rawurlencode($_GET['contactid']) : '')
-					. (!empty($_GET['search'])    ? '&search='    . rawurlencode($_GET['search'])    : '')
-					. (!empty($_GET['star'])      ? '&star='      . rawurlencode($_GET['star'])      : '')
-					. (!empty($_GET['order'])     ? '&order='     . rawurlencode($_GET['order'])     : '')
-					. (!empty($_GET['bmark'])     ? '&bmark='     . rawurlencode($_GET['bmark'])     : '')
-					. (!empty($_GET['liked'])     ? '&liked='     . rawurlencode($_GET['liked'])     : '')
-					. (!empty($_GET['conv'])      ? '&conv='      . rawurlencode($_GET['conv'])      : '')
-					. (!empty($_GET['nets'])      ? '&nets='      . rawurlencode($_GET['nets'])      : '')
-					. (!empty($_GET['cmin'])      ? '&cmin='      . rawurlencode($_GET['cmin'])      : '')
-					. (!empty($_GET['cmax'])      ? '&cmax='      . rawurlencode($_GET['cmax'])      : '')
-					. (!empty($_GET['file'])      ? '&file='      . rawurlencode($_GET['file'])      : '')
+					. (!empty($_GET['search']) ? '&search=' . rawurlencode($_GET['search']) : '')
+					. (!empty($_GET['star']) ? '&star=' . rawurlencode($_GET['star']) : '')
+					. (!empty($_GET['order']) ? '&order=' . rawurlencode($_GET['order']) : '')
+					. (!empty($_GET['bmark']) ? '&bmark=' . rawurlencode($_GET['bmark']) : '')
+					. (!empty($_GET['liked']) ? '&liked=' . rawurlencode($_GET['liked']) : '')
+					. (!empty($_GET['conv']) ? '&conv=' . rawurlencode($_GET['conv']) : '')
+					. (!empty($_GET['nets']) ? '&nets=' . rawurlencode($_GET['nets']) : '')
+					. (!empty($_GET['cmin']) ? '&cmin=' . rawurlencode($_GET['cmin']) : '')
+					. (!empty($_GET['cmax']) ? '&cmax=' . rawurlencode($_GET['cmax']) : '')
+					. (!empty($_GET['file']) ? '&file=' . rawurlencode($_GET['file']) : '')
```

Comment:
What has your IDE dooooone?

Reply:
Not my IDE, the automated check disliked it.